grammar.coffee

```diff
@@ -195,6 +205,14 @@ module.exports = grammar
 			choice(@_loop_statement, @if_statement)
 		)
 
+		break_statement: -> prec(PREC.BREAK_STATEMENT, seq('break', optional(@identifier)))
+
+		continue_statement: -> prec(PREC.CONTINUE_STATEMENT, seq('continue', optional(@identifier)))
+
+		fallthrough_statement: -> 'fallthrough'
+
+		return_statement: -> prec(PREC.RETURN_STATEMENT, seq('return', optional(@_expression)))
```

Comment:
Ok, I have a fix that will make these tests pass if you recompile. Just need to run it through CI and then I'll publish a new version of `tree-sitter-compiler`.

With the fix, you should be able to just use `choice(';', '\n')` at the end of the `_statement` rule, since any subsequent line feeds will just parse as part of your ubiquitous `/s+/` rule.